setup.cfg

```diff
@@ -49,22 +49,18 @@ install_requires =
     html5lib
     nbconvert
     nbformat
-    pytest >= 4.6
+    pytest >= 7.0
```

Comment:
No, because they changed their API to accept `path` instead of `fspath`.  We could keep `fspath` and ignore the deprecation warning, but we'd get caught when they remove `fspath` altogether.

Reply:
welp, 7 came out like last month... i daresay a lot of folk are stuck in the near term on 6: i've seen some pinning to that effect. it seems like being able to support the end of the 6 line would be worth some inconvenience if it were possible... as it's constructor stuff, i suppose _our_ constructor could accept both, and then forward on the correct one, but yeah, that's rather a lot of work for a test utility package.

but i guess as long as it's well-marked, on we go to `0.7.0`!